packages/cli/test/platformatic.test.js

```diff
@@ -21,7 +21,7 @@ test('version', async (t) => {
 
 test('db', async (t) => {
   try {
-    await execa('node', [cliPath, 'db'])
+    await execa('node', [cliPath, 'db', 'start'])
```

Comment:
`platformatic db` without any subcommand should start the app. But IMHO, it would be better to print help. So, I'm ok with this change. WDYT @mcollina @simonplend?

Reply:
It was already marked as a bug in https://github.com/platformatic/platformatic/issues/450 that's why I altered the behavior here :)

Reply:
Yes, I think this change is fine, but the `platformatic db` command should still do _something_. I'm +1 on it printing the help text for the `db` command.

Reply:
I agree

Reply:
That's exactly how it works with change now :)

<img width="489" alt="image" src="https://user-images.githubusercontent.com/1523305/207258655-51655bb4-e719-4050-9cb9-eaa9571dfac5.png">